build/index.asset.php

```diff
@@ -0,0 +1 @@
+<?php return array('dependencies' => array('react', 'wp-components', 'wp-element', 'wp-polyfill'), 'version' => 'adf49c3b45f9ee2c8230');
```

Comment:
Now that there is a compiled code for the plugin, is it common to include the build inside the repo, or should that be done another way?

Reply:
The follow up PR: https://github.com/WordPress/create-block-theme/pull/131